src/app/client/js/controllers.js

```diff
@@ -196,48 +305,70 @@
     var family = '';
     var familyHover = '';
     var showVSTSChoices = false;
-    var selectedVcs = '';
+    var showCtfChoices = false;
+    var selectedButton = '';
 
-    $scope.vstsSelection = function (value) {
+    $scope.subFamilySelection = function (value) {
       $scope.familySelect(value);
     };
 
-    var setFamilyWhenNotVsts = function setFamilyWhenNotVsts(Vcs) {
-      if (Vcs != "VSTS") {
-        $scope.familySelect(Vcs);
-      }
-    };
-
-    var alwaysToggleWhenVsts = function alwaysToggleWhenVsts(Vcs) {
+    var alwaysToggleWhenWithSubfamily = function alwaysToggleWhenWithSubfamily(Vcs) {
```

Comment:
I know the original code had uppercase V in `Vcs`, but that's kind of odd. I feel like this should be `vcs`.